mix.exs

```diff
@@ -36,8 +36,27 @@ defmodule Membrane.Template.Mixfile do
 
   defp deps do
     [
-      {:membrane_core, "~> 0.7.0"},
-      {:ex_doc, "~> 0.24", only: :dev, runtime: false},
+      {:membrane_core, "~> 0.8.0"},
```

Comment:
Here are tons of deps, can you group them by e.g. `only: :test` etc.? Separate them by commetns.